src/confdb/converter/python/PythonTaskWriter.java

```diff
@@ -6,10 +6,20 @@
 import confdb.converter.ITaskWriter;
 import confdb.data.Reference;
 import confdb.data.Task;
+import confdb.data.ReleaseVersionInfo;
 
 public class PythonTaskWriter implements ITaskWriter {
 	public String toString(Task task, ConverterEngine converterEngine, String object) {
-		String str = object + task.name() + " = cms.Task( ";
+		
+		ReleaseVersionInfo relInfo = new ReleaseVersionInfo(task.config().releaseTag());
+		String taskType = null;
+		if(relInfo.geq(12,4,0,4)){
```

Comment:
```suggestion
		if(relInfo.geq(12,4,0) && !(relInfo.equals(12,5,0,1))){
```
(the code is just an example, I leave the implementation to Sam)

In light of the issue spotted by @Sam-Harper in `12_4_0_pre4`, we should probably enable `ConditionalTask`s only starting from `12_5_0_pre2` and `12_4_0`.

The reason is that https://github.com/cms-sw/cmssw/pull/38015 hadn't made into `12_4_0_pre4` (and `12_5_0_pre1` is effectively the same as `12_4_0_pre4`). Some details on this can be found in https://github.com/cms-sw/cmssw/issues/36938#issuecomment-1150908993 and related comments.

Thanks to @Sam-Harper and @fwyzard for clarifying all this.

With this change, I think we could still target the new ConfDB release by next Monday, before the menus are migrated to `12_4_0` (which is expected to happen at some point next week).